recipes/dash-core-components/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "dash-core-components" %}
+{% set filename = "dash_core_components" %}
+{% set version = "0.22.1" %}
+{% set sha256 = "2dbbc0121eab102d92f7b407099cb28818676f3753ea635fb29a9f8b9df0a118" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ filename }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
The new conda-forge standard with python packages is to use `pip`:

https://conda-forge.org/docs/meta.html#use-pip